sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpPlugin.java

```diff
@@ -92,9 +92,10 @@ private static void addPhpUnitExtensions(Context context) {
     context.addExtensions(PhpUnitSensor.class,
       PropertyDefinition.builder(PhpUnitSensor.PHPUNIT_TESTS_REPORT_PATH_KEY)
```

Comment:
I thought about renaming the property (`path` -> `paths`) or introducing a new one, but it seems like to much. I also looked at `sonar-python`, and they have a singular noun in property, but it allows for multiple files.